packages/create-remix/rollup.config.js

```diff
@@ -0,0 +1,52 @@
+/* eslint-disable import/no-extraneous-dependencies */
+const path = require("path");
+const babel = require("@rollup/plugin-babel").default;
+const nodeResolve = require("@rollup/plugin-node-resolve").default;
+const copy = require("rollup-plugin-copy");
+
+const {
+  buildDir,
+  copyToPlaygrounds,
+  createBanner,
+  executableBanner,
+  getVersion,
+} = require("../../rollup.utils");
+
+/** @returns {import("rollup").RollupOptions[]} */
+module.exports = function rollup() {
+  let sourceDir = path.relative(process.cwd(), __dirname) || ".";
```

Comment:
This changes from what used to be a static `packages/{name}`.  We now need `sourceDir` to be relative to the current working directory, so that configs work fine regardless of whether they're run from the root or the individual `packages/{name}` folder